.github/workflows/e2e.yml

```diff
@@ -61,6 +62,15 @@ jobs:
           test-cloud-shell:$GITHUB_SHA
       env:
         PROJECT_ID: ${{ secrets.E2E_PROJECT_ID  }}
+    - name: Run Provisioning Test
+      run: |
+        set -x
+        # install dependencies
+        python3 -m pip install -r tests/provisioning/requirements.txt
+        # run tests
+        python3 tests/provisioning/provision_test.py $PROJECT_ID
```

Comment:
Consider running the tests in docker. This way when the finish, you don't need to  cleanup the state of the github worker VM - you just remove the docker container.